schedule/sles4sap/sap_deployment_automation_framework/ensa2_hanadb_sapcontrol_move_ascs.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: sap_deployment_automation_framework
+description: |
+  ENSA2 Central Services with HANA DB - Use sapcontrol to move ASCS
+  test scenario executed on deployment created by 'SAP Deployment automation framework'.
+  Test variant connects to SUT directly through ssh proxy jump, which allows direct use of OpenQA API calls.
+  This should allow to use standard sles4sap/HA libraries.
+vars:
+  TEST_CONTEXT: 'OpenQA::Test::RunArgs'
+schedule:
+  - boot/boot_to_desktop
+  - sles4sap/sap_deployment_automation_framework/connect_to_deployer
+  - sles4sap/sap_deployment_automation_framework/prepare_ssh_config
+  - sles4sap/redirection_tests/redirection_check
+  - sles4sap/redirection_tests/hana_cluster_check
```

Comment:
I am wondering if cluster check should be here. 
Technically you could execute this scenario on top of single HANA DB and in that case the test would fail while checking the cluster. 
Maybe it could be conditional schedule.

Reply:
Added conditional schedule.  See "New VRs:" on the top.